### PR TITLE
Add RT tests #938

### PR DESCRIPTION
**Pull request by dzedro**

### CODE REVIEW

**Comment by okurz** [comment]
+1

File: main.pm [hunk 1]
```diff
@@ -480,6 +480,9 @@ sub load_zdup_tests() {
 
 sub load_consoletests() {
     if (consolestep_is_applicable) {
+        if (check_var("ADDONS", "rt,ha,geo")) {
```
Comment (sysrich) [inline main.pm]: shouldn't this be 'if ADDONS *includes* rt'? or does the kmp modules test ONLY make sense if using RT with HA and GEO?

Comment (dzedro) [inline main.pm]: I'm not sure, there is some reason why is this done only with rt,ha,geo, I will ask

Comment (dzedro) [inline main.pm]: includes rt won't hurt

**Pull request merged**